operations/services.md

```diff
@@ -1089,8 +1101,8 @@ Returns prices of a rate in the specified interval. Note that response contains
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `RateId` | string | required | Unique identifier of the [Rate](services.md#rate) whose prices should be returned. |
-| `StartUtc` | string | required, max length 3 months | Start of the interval in UTC timezone in ISO 8601 format. |
-| `EndUtc` | string | required, max length 3 months | End of the interval in UTC timezone in ISO 8601 format. |
+| `FirstTimeUnit` | [TimeUnit](#time-unit) | required | First TimeUnit of the interval. |
```

Comment:
There as max length 3 months mentioned here. Do not we want to keep mentioning it somehow? Maybe like required, max length - where this max length would be the link to some general max length paragraph describing the limitation of 100 units or 2 years? cc @vitasamek